exporter/exporterhelper/queued_retry_test.go

```diff
@@ -56,7 +56,7 @@ func TestQueuedRetry_DropOnPermanentError(t *testing.T) {
 	})
 
 	ocs.run(func() {
-		// This is asynchronous so it should just enqueue, no errors expected.
+		// This is asynchronous so it should just enqueue, no errors expectedError.
```

Comment:
Is this change and other similar changes below intentional?

Reply:
Looks like an overzealous replace, I'll fix it.